.github/workflows/core.yml

```diff
@@ -37,13 +37,16 @@ jobs:
         integration: [magento, shopware, px]
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 18
       - run: |
-          yarn create storefront-x --directory=app --integration=${{ matrix.integration }}
-          cd app
           yarn install
+          node modules/create-storefront-x --directory=app --integration=${{ matrix.integration }}
+          cd app
+          > yarn.lock
+          YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn install
```

Comment:
This will still download modules from NPM. Just using local version of create script is not enough. You should be able to go through this whole testing pipeline on your local machine without internet, which would show this yarn install failing because of yarn trying to connect to NPM.